mono/metadata/class-internals.h

```diff
@@ -1447,6 +1447,9 @@ mono_class_contextbound_bit_offset (int* byte_offset_out, guint8* mask_out);
 gboolean
 mono_class_init_checked (MonoClass *klass, MonoError *error);
 
+gboolean
+mono_class_init_internal (MonoClass *klass);
```

Comment:
Please move this to `class-init.h` - all class setup/initialization entrypoints ought to go there.